accounts/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
```

Comment:
is it require to add all this dependencies in each module?

Reply:
removed unnecessary